package.json

```diff
@@ -65,5 +65,10 @@
   },
   "dependencies": {
     "single-spa": ">= 5.1.1"
+  },
+  "prettier": {
```

Comment:
Same comment on the other PR - it doesn't make sense to me to add a prettier config unless we're actually using prettier in the project.

Reply:
True. Left a comment regarding this in another PR.

Reply:
👍 I'll merge this in as-is and then the PR with prettier can be added later.

Reply:
Removed it :slightly_smiling_face:

Reply:
Oops 😄